### PR TITLE
Feat: add strip and radius circle draw

### PR DESCRIPTION
**Pull request by mbarbet**

### CODE REVIEW

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.direct.mode.ts [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    }
+    if(!state.start || state.selectedResizePaths !== state.selectedCoordPaths[0] ){
+        if (state.selectedCoordPaths[0] === '0.0') {
+            start = state.actionWidgets.find(a => a.properties.coord_path === '0.2').properties.coord_path_coords;
+        } else {
+            start = state.actionWidgets.find(a => a.properties.coord_path === '0.0').properties.coord_path_coords;
+        }
+        state.start = start;
+    }
+    const dist = distance(point(state.start,), point(end),{units:'kilometers'});
```
Comment (sebbousquet) [inline projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.direct.mode.ts:304]: Something strange in `point()` parameters, I think there is an extra comma

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.direct.mode.ts [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+import rhumbBearing from '@turf/rhumb-bearing';
+import { buildStrip } from './strip.mode';
+
+export const stripDirectSelectMode: any = {};
+
+
+stripDirectSelectMode.onSetup = function (opts) {
+    const featureId = opts.featureId;
+    const halfSwath = opts.halfSwath;
+    const maxLenght =opts.maxLenght;
```
Comment (sebbousquet) [inline projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.direct.mode.ts:18]: Missing space after `=`

**Review by sebbousquet — APPROVED** [review]

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.direct.mode.ts [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+            mutate: false,
+        });
+    state.start = undefined;
+    state.feature.incomingCoords(rotatedFeature.geometry.coordinates);
+    this.fireUpdate();
+};
+
+
+stripDirectSelectMode.dragResizePoint = function (state, e, delta) {
+    const end =[e.lngLat.lng, e.lngLat.lat];
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.direct.mode.ts:291]: missing space after =

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.mode.ts [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+    const corner1 = rhumbDestination(startPoint, halfSwath, bearing - 90);
+    const corner2 = rhumbDestination(startPoint, halfSwath, bearing + 90);
+    const corner3 = rhumbDestination(endPoint, halfSwath, bearing + 90);
+    const corner4 = rhumbDestination(endPoint, halfSwath, bearing - 90);
+    coordinates.push(corner1.geometry.coordinates);
+    coordinates.push(corner2.geometry.coordinates);
+    coordinates.push(corner3.geometry.coordinates);
+    coordinates.push(corner4.geometry.coordinates);
+    coordinates.push(coordinates[0]);
+    properties.start=start;
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.mode.ts:54]: missing space around =

File: projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.mode.ts [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+    if (stripLenght <= state.maxLenght) {
+        const stripFeature = buildStrip(start, end, state.halfSwath);
+        stripFeature.properties.parent = state.line.id;
+        (stripFeature.properties as any).meta = 'strip';
+        state.strip.setCoordinates(stripFeature.geometry.coordinates);
+        const startPoint = point(start);
+        const endPoint = point(end);
+        const bearing = rhumbBearing(startPoint, endPoint);
+        state.currentMaxBearing = bearing;
+    }else{
```
Comment (QuCMGisaia) [inline projects/arlas-components/src/lib/components/mapgl/draw/modes/strip/strip.mode.ts:161]: missing spaces around else

**Review by QuCMGisaia — APPROVED** [review]

**Pull request merged**